cmd/kubectl-gadget/advise/seccomp-policy.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 var seccompAdvisorCmd = &cobra.Command{
-	Use:   "seccomp-advisor",
+	Use:   "seccomp-policy",
```

Comment:
This should actually be seccomp-profile, as that's what the object generated is actually called